hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/datanode/UsageInfoSubcommand.java

```diff
@@ -155,6 +155,8 @@ private void printInfo(DatanodeUsage info) {
         + " B", StringUtils.byteDesc(info.getRemaining()));
     System.out.printf("%-13s: %s %n", "Remaining %",
         PERCENT_FORMAT.format(info.getRemainingRatio()));
+    System.out.printf("%-13s: %s (%s) %n", "Committed", info.getCommitted()
```

Comment:
`Committed` doesn't seem to be easy to understand outside of the code context, suggesting: `Container Preallocated`. 
And add a `Remaining Allocatable` (remaining - commited)